src/index.tsx

```diff
@@ -703,6 +708,23 @@ ReactDiffViewerState
         top: this.renderDiff(this.renderLeftView),
         bottom: this.renderDiff(this.renderRightView),
       };
+      if (returnDiffNodesArray) {
+        return [
```

Comment:
not sure I understand the purpose of this change, why not using a fragment for that?